app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dropzone from 'react-dropzone';
+import AttachmentsList from '../attachment/AttachmentList';
+import { upload, acceptedMimeTypes } from '../../../models/beta/fileUpload';
+
+class ExpandedStoryAttachments extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: false
+    }
+
+    this.onFileDrop = this.onFileDrop.bind(this);
+  }
+
+  onFileDrop(files) {
+    const newFile = files[0];
```

Comment:
What you think about writing this line as `const [newFile] = files;`?

Reply:
💅

Reply:
OMG, this is so fancy! :smile: